src/internal/modules/invoice-accounts/lib/helpers.js

```diff
@@ -11,11 +13,15 @@ const getFormTitleCaption = (licenceNumber) => {
 const processCompanyFormData = (request, regionId, companyId, formData) => {
   const { selectedCompany, companySearch } = forms.getValues(formData);
   if (selectedCompany === 'company_search') {
-    // TODO place holder -- route does not exist
-    return `company-search?filter=${companySearch}`;
+    return `contact-search?filter=${companySearch}`;
```

Comment:
This could potentially lead to confusion if we call it contact search because you are basically searching for a company and not a contact in the database.

Reply:
The reason I changed the URL path is that this is visible to the end user - the language in the prototype says 'Contact' rather than 'Company'. Are you sure it should be `company-search`? Not a big deal I suppose - Happy to change it if you feel I should.